rep-2005.rst

```diff
@@ -0,0 +1,568 @@
+    * `fastrtps/fastrtps <https://index.ros.org/p/fastrtps/>`_
+    * `rmw_fastrtps/rmw_fastrtps_cpp <https://index.ros.org/p/rmw_fastrtps_cpp/>`_
+    * `rmw_fastrtps/rmw_fastrtps_dynamic_cpp <https://index.ros.org/p/rmw_fastrtps_dynamic_cpp/>`_
+    * `rmw_fastrtps/rmw_fastrtps_shared_cpp <https://index.ros.org/p/rmw_fastrtps_shared_cpp/>`_
+    * `rosidl_typesupport_fastrtps/fastrtps_cmake_module <https://index.ros.org/p/fastrtps_cmake_module/>`_
+    * `rosidl_typesupport_fastrtps/rosidl_typesupport_fastrtps_c <https://index.ros.org/p/rosidl_typesupport_fastrtps_c/>`_
+    * `rosidl_typesupport_fastrtps/rosidl_typesupport_fastrtps_cpp <https://index.ros.org/p/rosidl_typesupport_fastrtps_cpp/>`_
+
+  * OpenSplice
+
+    * `rmw_opensplice/rmw_opensplice_cpp <https://index.ros.org/p/rmw_opensplice_cpp/>`_
+    * `rosidl_typesupport_opensplice/opensplice_cmake_module <https://index.ros.org/p/opensplice_cmake_module/>`_
+    * `rosidl_typesupport_opensplice/rosidl_typesupport_opensplice_c <https://index.ros.org/p/rosidl_typesupport_opensplice_c/>`_
+    * `rosidl_typesupport_opensplice/rosidl_typesupport_opensplice_cpp <https://index.ros.org/p/rosidl_typesupport_opensplice_cpp/>`_
+
+* Client libraries
+
+  * `rcl/rcl <https://index.ros.org/p/rcl/>`_
+  * `rcl/rcl_action <https://index.ros.org/p/rcl_action/>`_
+  * `rcl/rcl_lifecycle <https://index.ros.org/p/rcl_lifecycle/>`_
+  * `rcl/rcl_yaml_param_parser <https://index.ros.org/p/rcl_yaml_param_parser/>`_
+  * `rcl_logging/rcl_logging_log4cxx <https://index.ros.org/p/rcl_logging_log4cxx/>`_
+  * `rcl_logging/rcl_logging_noop <https://index.ros.org/p/rcl_logging_noop/>`_
+  * `rcl_logging/rcl_logging_spdlog <https://index.ros.org/p/rcl_logging_spdlog/>`_
+  * `rclcpp/rclcpp <https://index.ros.org/p/rclcpp/>`_
+  * `rclcpp/rclcpp_action <https://index.ros.org/p/rclcpp_action/>`_
+  * `rclcpp/rclcpp_components <https://index.ros.org/p/rclcpp_components/>`_
+  * `rclcpp/rclcpp_lifecycle <https://index.ros.org/p/rclcpp_lifecycle/>`_
+  * `rclpy/rclpy <https://index.ros.org/p/rclpy/>`_
+
+* Orchestration
+
+  * `launch/launch <https://index.ros.org/p/launch/>`_
+  * `launch/launch_testing <https://index.ros.org/p/launch_testing/>`_
+  * `launch/launch_testing_ament_cmake <https://index.ros.org/p/launch_testing_ament_cmake/>`_
+  * `launch/launch_xml <https://index.ros.org/p/launch_xml/>`_
+  * `launch/launch_yaml <https://index.ros.org/p/launch_yaml/>`_
+  * `launch_ros/launch_ros <https://index.ros.org/p/launch_ros/>`_
+  * `launch_ros/launch_testing_ros <https://index.ros.org/p/launch_testing_ros/>`_
+  * `launch_ros/ros2launch <https://index.ros.org/p/ros2launch/>`_
+
+* Features
+
+  * `diagnostics/diagnostic_updater <https://index.ros.org/p/diagnostic_updater/>`_
+  * `diagnostics/self_test <https://index.ros.org/p/self_test/>`_
+  * `xacro/xacro <https://index.ros.org/p/xacro/>`_
+  * `robot_state_publisher/robot_state_publisher <https://index.ros.org/p/robot_state_publisher/>`_
```

Comment:
can we add [joint_state_publisher](https://github.com/ros/joint_state_publisher/tree/ros2) to this list ?

Reply:
Yeah, that's a key tool. I added both packages from that repo in 43008bb.